docs/sdks/dart/topics.mdx

```diff
@@ -87,5 +87,5 @@ This is an example of subscribing to a topic called "topic", then catching the r
 You can find complete, working examples in the [Dart SDK GitHub repo examples directory](https://github.com/momentohq/client-sdk-dart/tree/main/example).
 
 :::info
-Beyond these basic API calls check out the [API reference page](/develop/api-reference) for more information on the full assortment of Momento API calls.
+Beyond these basic API calls check out the [API reference page](/topics/develop/api-reference) for more information on the full assortment of Momento API calls.
```

Comment:
hrm, I could have sworn docusaurus was supposed to catch these. thanks for fixing.

Reply:
The latest parser is stricter about links and by default doesn't permit linking to a redirect. We have a redirect for `/develop/api-reference -> /cache/develop/api-reference`, so even though the link wasn't technically broken here, it was wrong. 😓